agent-control/src/agent_type/runtime_config/onhost.rs

```diff
@@ -134,7 +144,16 @@ mod tests {
                 max_retries: TemplateableValue::from_template("3".to_string()),
                 last_retry_interval: TemplateableValue::from_template("30s".to_string()),
             },
-            on_host.executable.unwrap().restart_policy.backoff_strategy
+            on_host.executables.clone().first().unwrap().restart_policy.backoff_strategy
+        );
+        assert_eq!(
+            BackoffStrategyConfig {
```

Comment:
they seems the same, could we do a loop or save BackoffStrategyConfig in a variable?

if hyst a field is changing we could just modify that to show the difference

Reply:
Done in last commit.